docs/index.md

```diff
@@ -14,8 +14,12 @@ Listen for queued jobs.
 
 ### Requirements
 
-![PHP](https://img.shields.io/badge/PHP-%5E8.1-red)
-![CodeIgniter](https://img.shields.io/badge/CodeIgniter-%5E4.3-red)
+- PHP 8.1+
+- CodeIgniter 4.3+
+
+If you use `database` handler:
+- MySQL 8.0.1+
+- MariaDB 10.6+
```

Comment:
```suggestion
- MariaDB 10.6+
- PostgreSQL 9.5+
- SQL Server 2005+
- Oracle 12.1+
```

Reply:
What about SQLite3?

It seems PHPUnit tests run only on SQLite.
These versions are actually checked?
Now CI4 officially supports SQL Server 2012+.
https://codeigniter4.github.io/CodeIgniter4/intro/requirements.html#supported-databases

Reply:
> What about SQLite3?

SQLite3 doesn't support `SKIP LOCKED` feature at all.

> It seems PHPUnit tests run only on SQLite.
These versions are actually checked?

No. I checked it only once - manually, except for Oracle. 

> Now CI4 officially supports SQL Server 2012+.

Updated.

Reply:
I don't understand the details, but Solid Queue supports SQLite.

> use_skip_locked: whether to use FOR UPDATE SKIP LOCKED when performing locking reads. This will be automatically detected in the future, and for now, you'd only need to set this to false if your database doesn't support it. For MySQL, that'd be versions < 8, and for PostgreSQL, versions < 9.5. If you use SQLite, this has no effect, as writes are sequential.
https://github.com/basecamp/solid_queue

Reply:
We also support SQLite, but there is no `FOR UPDATE SKIP LOCKED` in SQLite in general.

Reply:
Then, do we add `SQLite3` in the list?

Reply:
Yes, we should.